static/js/components/dashboard/FinancialAidCard.js

```diff
@@ -213,7 +214,7 @@ export default class FinancialAidCard extends React.Component {
           <Grid>
             <Cell col={1} />
             <Cell col={5}>Mail to</Cell>
-            <Cell col={6}>or fax</Cell>
+            <Cell col={6}>Upload to DocuSign</Cell>
```

Comment:
This acceptance criteria:

- [ ] Move the mail section to the right

means that the cells for mail and upload should be swapped. We want the upload on the left and the mail on the right.

Reply:
updated.